joint_limits:
  wheel_left_joint:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 4.2
    has_acceleration_limits: false
    max_acceleration: 10.0
    has_jerk_limits: false
    max_jerk: 100.0

  wheel_right_joint:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 4.2
    has_acceleration_limits: false
    max_acceleration: 10.0
    has_jerk_limits: false
    max_jerk: 100.0
